### PR TITLE
Feature 806 screening filters and display on dashboard/activity feeds

### PR DESCRIPTION
**Pull request by emullaney**
https://github.com/OpenTechFund/opentech.fund/issues/806, requirements copied below:
- Submission detail page will be updated to include the outcome of the screening in the sidebar (as per determination block, placed above determination block)
- Screening block on submission detail page only visible to OTF staff and admin
- 'Screening' to be added as a column in submissions table to display outcome of screening (currently called remit in designs)
- 'Screening' to be added as filter on submissions table
- Screening activity will be visible in the activity feed for OTF Staff
- This taxonomy is not visible to the applicant at any time
- Added unit tests

### CODE REVIEW

**Comment by emullaney** [comment]
I've added some unit tests, can you please give this a review @todd-dembrey ? I'm not 100% sure I covered everything, so please let me know if you catch anything. Thanks!

File: opentech/apply/activity/messaging.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def recipients(self, message_type, **kwargs):
         return [None]
 
     def extra_kwargs(self, message_type, **kwargs):
-        if message_type in [MESSAGES.OPENED_SEALED, MESSAGES.REVIEWERS_UPDATED]:
+        if message_type in [MESSAGES.OPENED_SEALED, MESSAGES.REVIEWERS_UPDATED, MESSAGES.SCREENING]:
```
Comment (todd-dembrey) [inline opentech/apply/activity/messaging.py]: The following issue is happening on the site.
```
2019-01-15T17:35:19.901570+00:00 app[web.1]: [2019-01-15 17:35:19,897] loader_tags: WARNING - Exception raised while rendering {% include %} for template 'funds/applicationsubmission_admin_detail.html'. Empty string rendered instead.
...
2019-01-15T17:35:19.901678+00:00 app[web.1]: AttributeError: Cannot find 'author' on ScreeningStatus object, 'related_object__author' is an invalid parameter to prefetch_related()
```

We save the related object on the Activity when it is a model this allows us to link to it within the activity objects. You need to either pass `str(old.screening_status)` from https://github.com/OpenTechFund/opentech.fund/blob/57e9efc58c69e358c3183dded4ba3184034c19b3/opentech/apply/funds/views.py#L130

We use the related object to link to objects with pages, screening status doesn't have a page.

Relevant code.
https://github.com/OpenTechFund/opentech.fund/blob/57e9efc58c69e358c3183dded4ba3184034c19b3/opentech/apply/activity/models.py#L86-L89

https://github.com/OpenTechFund/opentech.fund/blob/57e9efc58c69e358c3183dded4ba3184034c19b3/opentech/apply/activity/templates/activity/include/listing_base.html#L25-L31

Comment (emullaney) [reply opentech/apply/activity/messaging.py]: Ok, I think you outlined a simple solution here, switching out 
`related=str(old.screening_status)` 
for 
`related=old.screening_status or '-'`

**Comment by emullaney** [comment]
@todd-dembrey I think I made the change you requested. I didn't see that exact error myself so I'm not sure how to test it but if I missed the mark, please let me know. Thanks!!

**Pull request merged**